Source/Fuse.Storage/Json.uno

```diff
@@ -0,0 +1,108 @@
+using Uno;
+using Uno.Data.Json;
+using Uno.Collections;
+
+namespace Fuse.Storage
+{
+	public static class Json
+	{
+		internal static object Parse(string json)
+		{
+			return Convert(JsonReader.Parse(json));
+		}
+
+		internal static object[] Parse(string[] json)
+		{
+			var res = new object[json.Length];
+			for (int i = 0; i < res.Length; i++)
+			{
+				res[i] = Parse(json[i]);
+			}
+			return res;
+		}
+
+		internal static string Stringify(object value)
+		{
+			return Fuse.Json.Stringify(value, false);
+		}
+
+		internal static string Stringify(Dictionary<string, object> value)
+		{
+			return Fuse.Json.Stringify(new Object(value), false);
+		}
+
+		static object Convert(JsonReader r)
+		{
+			switch (r.JsonDataType)
+			{
+				case JsonDataType.Object:
+					{
+						var keys = r.Keys;
+						var dict = new Dictionary<string, object>();
+						for (int i = 0; i < keys.Length; i++)
+							dict.Add(keys[i], Convert(r[keys[i]]));
+						return new Object(dict);
+					}
+				case JsonDataType.Array:
+					{
+						var values = new object[r.Count];
+						for (int i = 0; i < values.Length; i++)
+							values[i] = Convert(r[i]);
+						return new Array(values);
+					}
+				case JsonDataType.Number: return r.AsNumber();
+				case JsonDataType.String: return (string)r;
+				case JsonDataType.Boolean: return (bool)r;
+			}
+			return null;
+		}
+
+		internal class Object : IObject
+		{
+			readonly Dictionary<string, object> _dict;
+
+			public Object(Dictionary<string, object> dict)
+			{
+				_dict = dict;
+			}
+
+			public string[] Keys
+			{
+				get { return _dict.Keys.ToArray(); }
+			}
+
+			public bool ContainsKey(string key)
+			{
+				return _dict.ContainsKey(key);
+			}
+
+			public object this[string key]
+			{
+				get
+				{
+					return _dict[key];
+				}
+			}
+		}
+
+		class Array: IArray
+		{
+			readonly object[] _array;
+
+			public Array(params object[] array)
+			{
+				_array = array;
+			}
+
+			public int Length
+			{
+				get { return _array.Length; }
+			}
+
+			public object this[int index]
+			{
+				get { return _array[index]; }
+			}
+		}
+	}
+}
```

Comment:
What makes this different from [Fuse.Json](https://github.com/fuse-open/fuselibs/blob/master/Source/Fuse.Common/Json.Parse.uno#L7)? Why can't we just reuse that one?

Reply:
Yeah, it is same, I created new Json.uno in `Fuse.Storage` because I cannot access the `parse` method in `Json.Parse.uno` in `Fuse.Common` because its modifier is `internal`.  I'm eager to change the method modifier in [Json.Parse.uno](https://github.com/fuse-open/fuselibs/blob/5f128ad3d6577b4b5faf8dc4c80452f76ea151ff/Source/Fuse.Common/Json.Parse.uno#L13) to `public` but I'm afraid. If it's OK to change its modifier, I will use that class instead and remove `Json.uno` on this PR

Reply:
Right. I think it's just a matter of adding this project to the [InternalsVisibleTo-list in Fuse.Common.unoproj](https://github.com/fuse-open/fuselibs/blob/5f128ad3d6577b4b5faf8dc4c80452f76ea151ff/Source/Fuse.Common/Fuse.Common.unoproj#L5), and you should be able to use it without copying.

Reply:
Adding to `InternalsVisibleTo` is a less drastic change because it's not changing public interfaces, but I'm for making the `Parse` method `public` instead, as @ichan-mb is eager to do. That seems easier and a bit cleaner to me, unless doing this ends up with more build errors because you'll need more internal stuff...

I see the `Parse` method is already documented and sounds like it may be useful for others to find...

In any case, this is up to you guys, just wanted to chime in with my opinion.

Reply:
Oh wow!, a new knowledge for me about `InternalVisibleTo` I noted that. Yeah I think I will make the `Parse` method `public` since I see `Stringify` method is `public` too [here](https://github.com/fuse-open/fuselibs/blob/5f128ad3d6577b4b5faf8dc4c80452f76ea151ff/Source/Fuse.Common/Json.uno#L15). It will make a complete pair of Json Function. Hope doing that doesn't make build errors